ci-scripts/copy_to_archive

```diff
@@ -54,6 +69,7 @@ if [ -d ${IMAGE_DIR}/${ARP_ARCH} ]; then
     mv ${IMAGE_DIR}/${ARP_ARCH}/${IMAGE_BASENAME}-*rootfs.wic.bmap $ARCHIVE_DIR
     mv ${IMAGE_DIR}/${ARP_ARCH}/${IMAGE_BASENAME}-*rootfs.wic.bz2 $ARCHIVE_DIR
     mv ${IMAGE_DIR}/${ARP_ARCH}/${IMAGE_BASENAME}-*20*.swu $ARCHIVE_DIR
```

Comment:
Consider to check if the swu-file is present, and in this case move it? What if the "update target" is not built? Should this step fail then?

Can it also be independent of variant, and applicable for any CPU architecture?